Pana crashes when package:lints is not found locally.

I believe this happens due to the recent changes in [lib/src/analysis_options.dart](https://github.com/dart-lang/pana/commit/7d702a5034f5d5dbf5add7bcf851a46438f9a066#diff-fae9cda8bacb4fee8c63068dd6cc9b313ee0488572c6be83c5d5f26cb675958a).

If the `resource` in Line 72 cannot be loaded, it is null. Line 74 then throws an `Error` (not an `Exception`!) which is not caught by line 77. Fix should be easy: Simply catch Errors and exceptions, or check if the resource is null.

Logs:
```
WARNING    Flutter SDK path was not specified, pana will use the default Dart SDK to run `dart analyze` on Flutter packages.
INFO       Running `/opt/hostedtoolcache/dart/3.0.5/x64/bin/dart --version`...
INFO       Running `flutter --no-version-check --version --machine`...
WARNING    Unable to detect Flutter version.
           ProcessException: No such file or directory
             Command: flutter --no-version-check --version --machine
WARNING    pana might update or modify files in `.`.
           Analysis will begin in 15 seconds, hit CTRL+C to abort it.
           To remove this message, use `--no-warning`.
INFO       Running `git rev-parse --show-toplevel`...
INFO       Running `/opt/hostedtoolcache/dart/3.0.5/x64/bin/dart pub get --no-example`...
INFO       Running `/opt/hostedtoolcache/dart/3.0.5/x64/bin/dart pub outdated --json --up-to-date --no-dev-dependencies --no-dependency-overrides`...
INFO       Analyzing package...
SHOUT      Problem analyzing .
           Null check operator used on a null value
           #0      _getLintsCoreAnalysisOptions (package:pana/src/analysis_options.dart:74:39)
           <asynchronous suspension>
           #1      getDefaultAnalysisOptionsYaml (package:pana/src/analysis_options.dart:29:12)
           <asynchronous suspension>
           #2      ToolEnvironment.runAnalyzer (package:pana/src/sdk_env.dart:1[8](https://github.com/Skycoder42/dart_test_tools/actions/runs/5352836295/jobs/9708217294#step:15:9)7:[9](https://github.com/Skycoder42/dart_test_tools/actions/runs/5352836295/jobs/9708217294#step:15:10))
           <asynchronous suspension>
           #3      PackageContext._staticAnalysis (package:pana/src/package_context.dart:190:22)
           <asynchronous suspension>
           #4      PackageContext.staticAnalysis (package:pana/src/package_context.dart:175:23)
           <asynchronous suspension>
           #5      PackageAnalyzer._inspect (package:pana/src/package_analyzer.dart:231:27)
           <asynchronous suspension>
           #6      PackageAnalyzer.inspectDir.<anonymous closure> (package:pana/src/package_analyzer.dart:146:14)
           <asynchronous suspension>
           #7      withTempDir (package:pana/src/utils.dart:236:[12](https://github.com/Skycoder42/dart_test_tools/actions/runs/5352836295/jobs/9708217294#step:15:13))
           <asynchronous suspension>
           #8      main (file:///home/runner/.pub-cache/hosted/pub.dev/pana-0.21.33/bin/pana.dart:[17](https://github.com/Skycoder42/dart_test_tools/actions/runs/5352836295/jobs/9708217294#step:15:18)6:[19](https://github.com/Skycoder42/dart_test_tools/actions/runs/5352836295/jobs/9708217294#step:15:20))
           <asynchronous suspension>
Error: Process completed with exit code 1.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 0.21.34
+
+- Export missing `PanaProcessResult`.
+
 ## 0.21.33
 
 - Do not block repository check when `pubspec.yaml` is in path that is not accepted.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,7 @@
 ## 0.21.34
 
 - Export missing `PanaProcessResult`.
+- Fix `package:lints` as main dependency.
 
 ## 0.21.33
 

--- a/lib/pana.dart
+++ b/lib/pana.dart
@@ -11,5 +11,5 @@ export 'src/pubspec.dart';
 export 'src/report/create_report.dart' show renderSimpleSectionSummary;
 export 'src/sdk_env.dart';
 export 'src/tag/pana_tags.dart';
-export 'src/utils.dart' show runProc, yamlToJson;
+export 'src/utils.dart' show runProc, yamlToJson, PanaProcessResult;
 export 'src/version.dart';

--- a/lib/src/screenshots.dart
+++ b/lib/src/screenshots.dart
@@ -6,7 +6,6 @@ import 'dart:io';
 import 'dart:typed_data';
 
 import 'package:pana/pana.dart';
-import 'package:pana/src/utils.dart';
 import 'package:path/path.dart' as path;
 import 'package:pubspec_parse/pubspec_parse.dart' as p;
 

--- a/lib/src/version.dart
+++ b/lib/src/version.dart
@@ -1,2 +1,2 @@
 // Generated code. Do not modify.
-const packageVersion = '0.21.34-dev';
+const packageVersion = '0.21.34';

--- a/lib/src/version.dart
+++ b/lib/src/version.dart
@@ -1,2 +1,2 @@
 // Generated code. Do not modify.
-const packageVersion = '0.21.33';
+const packageVersion = '0.21.34-dev';

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,6 +1,6 @@
 name: pana
 description: PAckage aNAlyzer - produce a report summarizing the health and quality of a Dart package.
-version: 0.21.34-dev
+version: 0.21.34
 repository: https://github.com/dart-lang/pana
 topics:
   - tool
@@ -19,6 +19,7 @@ dependencies:
   http: ^1.0.0
   io: ^1.0.0
   json_annotation: ^4.8.0
+  lints: ^2.0.0
   logging: ^1.0.0
   markdown: ^7.0.0
   path: ^1.6.2
@@ -40,7 +41,6 @@ dev_dependencies:
   build_verify: ^3.0.0
   build_version: ^2.0.0
   json_serializable: ^6.0.0
-  lints: ^2.0.0
   shelf: ^1.0.0
   source_gen: ^1.0.0
   test_descriptor: ^2.0.0

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,6 +1,6 @@
 name: pana
 description: PAckage aNAlyzer - produce a report summarizing the health and quality of a Dart package.
-version: 0.21.33
+version: 0.21.34-dev
 repository: https://github.com/dart-lang/pana
 topics:
   - tool

